src/main/resources/application.yml

```diff
@@ -8,8 +8,16 @@ spring:
       enabled: true
   jpa:
     hibernate:
-      ddl-auto: "create-drop" # FIXME: Just for early development without migrations
-
+      ddl-auto: "create" # FIXME: Just for early development without migrations
+    open-in-view: false
+    generate-ddl: true
+  data:
+    redis:
+      repositories:
+        enabled: false
+    jpa:
+        repositories:
+            enabled: true
```

Comment:
Do we need these lines. At least the default value of `spring.data.jpa.repositories.enabled` is true in Spring Boot.

Reply:
Since we are not directly using data-redis, I think it should be disabled (no need to scan for redis repos in startup)